cs251tk/student/markdownify/markdownify.py

```diff
@@ -7,12 +7,12 @@
 from ...common import check_dates
 
 
-def markdownify(spec_id, *, username, spec, basedir, debug, interact, student, web):
+def markdownify(spec_id, *, username, spec, basedir, debug, interact, student, web, ci):
     """Run a spec against the current folder"""
     try:
         first_submit = "First Submission for {}: {}".format(
             spec_id,
-            check_dates(spec_id, username, spec, basedir))
+            check_dates(spec_id, username, spec, basedir)) if not ci else ''
```

Comment:
Let's break this into multiple lines:

```suggestion
            check_dates(spec_id, username, spec, basedir))
        if ci:
            first_submit = ''
```

Reply:
I don't want it calling check_dates at all though because check_dates changes directories and isn't necessary for the ci.

Reply:
What about 
```
first_submit = ''

if not ci:
    first_submit = "First Submission for {}: {}".format(
    spec_id,
     check_dates(spec_id, username, spec, basedir))
```

Reply:
Sure. Can we pull the `check_dates` call into a separate variable, too, to reduce the line wrapping?

```python
first_submit = ''

if not ci:
    first_date = check_dates(spec_id, username, spec, basedir)
    first_submit = "First Submission for {}: {}".format(spec_id, first_date)
```